### DISCUSSION TITLE
Serious CSS Issues with Nuxt RC11

### DISCUSSION THREAD

**Discussion by duderion** [nuxt/nuxt#16555] [General]
Hi Nuxt Team,

with RC11 it looks like this

https://monosnap.com/file/SNRQH3y9b42DcCxpwmHlODVYxgGOTK

the unrendered code flashes and the style is cluttered if i reload the page.
inside the SPA, if i go forth and back, its correct.

the start page itself is a template with html and some external css files.

downgrading to RC9 fixed it.

**Comment by TheAlexLichter** [comment]
Possibly related: https://github.com/nuxt/framework/issues/8352
